classes/05class/exercises/c05-3m01/gepifanio/Makefile

```diff
@@ -0,0 +1,9 @@
+COMPOSE_RUN_AWS = docker-compose run --rm aws
+export BUCKET_NAME ?= c05-3m01
+
+create_bucket:
+	$(COMPOSE_RUN_AWS) sh ./scripts/create_bucket.sh
```

Comment:
One of the patterns is about splitting the high-level targets and technology-specific ones.
You used `sh` here, but any other shell or application can be executed. THat's why usually underscore targets are created, like `_ create_bucket`.

Please check:

https://3musketeers.io/docs/patterns.html#make